stubs/redis/redis/cluster.pyi

```diff
@@ -1,75 +1,80 @@
-from _typeshed import Incomplete
+from _typeshed import Incomplete, Self
 from collections.abc import Callable
 from threading import Lock
+from types import TracebackType
 from typing import Any, ClassVar, Generic
+from typing_extensions import Literal
 
-from redis.client import PubSub
-from redis.commands import RedisClusterCommands
+from redis.client import CaseInsensitiveDict, PubSub, Redis
+from redis.commands import CommandsParser, RedisClusterCommands
 from redis.commands.core import _StrType
-from redis.connection import BaseParser, Encoder
+from redis.connection import BaseParser, Connection, Encoder
 from redis.exceptions import RedisError
 
-def get_node_name(host, port): ...
-def get_connection(redis_node, *args, **options): ...
+def get_node_name(host: str, port: int) -> str: ...
```

Comment:
In the source code `port` is annotated as `Union[str, int]` -- any reason not to do that here? https://github.com/redis/redis-py/blob/c94821c11e74d360ae3ccdcf9581cfe24e120a07/redis/cluster.py#L41

```suggestion
def get_node_name(host: str, port: str | int) -> str: ...
```